sdk/src/store.rs

```diff
@@ -2072,8 +2072,19 @@ impl Store {
                             settings,
                         ) {
                             svi.timestamps.insert(rc.label().to_owned(), tst_info);
+                            continue;
```

Comment:
This section is a bad merge, the change made here should be undone.

Reply:
I removed these changes